src/components/story-element.js

```diff
@@ -94,12 +94,65 @@ const DEFAULT_TEMPLATES = {
 };
 
 class StoryElementBase extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.observer = null;
+    this.storyElementRef = null;
+  }
+
+  componentDidMount() {
+    this.initiateObserver();
+  }
+
+  componentWillUnmount() {
+    this.destroyObserver();
+  }
+
+  initiateObserver = () => {
+    if(this.props.disableAnalytics === true || (global && !global.qlitics)) return false;
```

Comment:
what if qlitics loads after the component has mounted?

Reply:
good point. I think you'll need to do the global.qlitics = global.qlitics || [];

qlitics.push()

Reply:
Added.